diff -Naur blobby-1.0.orig/src/NetworkMessage.cpp blobby-1.0/src/NetworkMessage.cpp
--- blobby-1.0.orig/src/NetworkMessage.cpp	2014-03-22 14:31:21.000000000 +0100
+++ blobby-1.0/src/NetworkMessage.cpp	2014-06-20 16:31:43.680681439 +0200
@@ -109,6 +109,6 @@
 
 std::ostream& operator<<(std::ostream& stream, const ServerInfo& val)
 {
-	return stream << val.name << " (" << val.hostname << ":" << val.port << ")";
+	return stream << std::string(val.name) << std::string(" (") << std::string(val.hostname) << std::string(":") << std::to_string(val.port) << std::string(")");
 }
 
