diff --git a/src/NetworkMessage.cpp b/src/NetworkMessage.cpp
index 9178e74..0c3163b 100644
--- a/src/NetworkMessage.cpp
+++ b/src/NetworkMessage.cpp
@@ -109,6 +109,11 @@ bool operator == (const ServerInfo& lval, const ServerInfo& rval)
 
 std::ostream& operator<<(std::ostream& stream, const ServerInfo& val)
 {
-	return stream << val.name << " (" << val.hostname << ":" << val.port << ")";
+	return stream
+                    << std::string(val.name)
+                    << std::string(" (")
+                    << std::string(val.hostname)
+                    << std::string(":") << std::to_string(val.port)
+                    << std::string(")");
 }
 
-- 
2.11.0

